packages/libs/components/src/stories/plotControls/TimeSlider.stories.tsx

```diff
@@ -303,6 +306,56 @@ export const TimeFilter: Story<LinePlotProps> = (args: any) => {
   const defaultSymbolSize = 0.8;
   const defaultColor = '#333';
 
+  // control selectedRange
+  const handleAxisRangeChange = useCallback(
+    (newRange?: NumberOrDateRange) => {
+      if (newRange)
+        setSelectedRange({
+          start: newRange.min as string,
+          end: newRange.max as string,
+        });
+    },
+    [setSelectedRange]
+  );
+
+  const handleArrowClick = useCallback(
```

Comment:
Ah, this is STORY code. I didn't understand how it worked but now I realise I should ignore it.